mobile_files/VERSION

```diff
@@ -1 +1 @@
-0.9.33
+190109-150228-d0d92a
```

Comment:
that looks... weird?

Reply:
mmh, that indeed looks weird, modified the wrong `VERSION` file :)

Reply:
fixed